arrows/core/video_input_image_list.cxx

```diff
@@ -426,4 +389,133 @@ ::metadata_map()
   return std::make_shared<kwiver::vital::simple_metadata_map>(d->m_metadata_map);
 }
 
+
+// ------------------------------------------------------------------
+void
+video_input_image_list::priv
+::read_from_file( std::string const& filename )
+{
+  typedef kwiversys::SystemTools ST;
+
+  // open file and read lines
+  std::ifstream ifs( filename.c_str() );
+  if ( ! ifs )
+  {
+    throw kwiver::vital::invalid_file( filename, "Could not open file" );
```

Comment:
The ``VITAL_THROW()`` macro is available to provide the actual source location where the exception is thrown from if you think it us useful. Some times knowing what file threw the exception can aid in debugging. This comment applies to all places where a kwiver::vital exception is thrown.

Reply:
This section of code was lifted directly above - I didn't write it. That said, do you think it's worth converting all of these to `VITAL_THROW()` as long as we're here?

Reply:
If you are editing the file and have the time, it would make the file more consistent.